netsync/manager.go

```diff
@@ -533,6 +533,23 @@ func (sm *SyncManager) handleBlockMsg(bmsg *blockMsg) {
 	}
 
 	if isOrphan {
+		blueScore, err := bmsg.block.BlueScore()
+		if err != nil {
+			log.Errorf("Received an orphan block %s with malformed blue score from %s. Disconnecting...",
+				blockHash, peer)
+			peer.Disconnect()
```

Comment:
Just reminding that this will have to be fixed once the Ban PR is merged. So we shouldn't merge until it is, and then this fixed.